rapcom/__main__.py

```diff
@@ -11,42 +11,44 @@
 import types
 
 from docopt import docopt
+import colorama
 import pkg_resources
+import six
 
 
 __all__ = ('main',)
 
 
+_logger = logging.getLogger(__name__)
+
+
 _COMMAND = os.path.basename(os.path.realpath(os.path.abspath(sys.argv[0])))
+_SUBCOMMANDS = {}
 _DEFAULT_DOC = """
-Usage: {command} <command> [<args>...]
-       {command} (-h | --help)
-       {command} (-V | --version)
+Usage:
+  {command} [--help] [--version] [--log-level <level>] <command> [<args>...]
 
 Options:
-  -h, --help           Display this help message and exit.
-  -V, --version        Display the version and exit.
+  --help               Display this help message and exit.
+  --version            Display the version and exit.
+  --log-level <level>  Set the log level to one of DEBUG, INFO, WARN, or ERROR.
 {{message}}
 '{command} help -a' lists all available subcommands.
 See '{command} help <command>' for more information on a specific command.
 """.format(command=_COMMAND)
 
 
-logging.getLogger().addHandler(logging.NullHandler())
-_logger = logging.getLogger(__name__)
-_subcommands = {}
-
-
 def main():
     """Parse the command line options and launch the requested command.
 
     If the command is 'help' then print the help message for the subcommand; if
     no subcommand is given, print the standard help message.
     """
-    _subcommands.update(_get_subcommands())
+    _SUBCOMMANDS.update(_get_subcommands())
     dist_version = _get_dist_version()
     doc = _DEFAULT_DOC.format(message='')
     args = docopt(doc, version=dist_version, options_first=True)
+    _enable_logging(args['--log-level'])
```

Comment:
If the log level isn't set, this sets a `logging.NullHandler`, which is the default.

I don't actually want logging enabled by default because the display widgets like `run_tasks` have to change their behavior to just log.. I'm not sure on how that's going to work yet, but it interferes with the their usual behavior if `stderr` isn't redirected.